core/src/main/java/io/undertow/server/handlers/GracefulShutdownHandler.java

```diff
@@ -83,9 +91,32 @@ private static long activeCount(long state) {
 
     @Override
     public void handleRequest(HttpServerExchange exchange) throws Exception {
+        if (exchange.getConnection() instanceof Http2ServerConnection) {
+            // Track Http2 channels to be able to perform the shutdown procedure
+            // defined in rfc7540 using GOAWAY frames.
```

Comment:
Please, rename rfc7540 to rfc9113, as this is the latest spec. Alternatively you can also add both specs to the comment